.dev-lib

```diff
@@ -38,6 +38,30 @@ function after_wp_install {
 		unzip -d "$WP_CORE_DIR/src/wp-content/plugins/pwa/" "$WP_CORE_DIR/src/wp-content/plugins/pwa.zip"
 		echo "done"
 	fi
+
+	if [[ $(php -r "echo PHP_VERSION;") == 8.0* ]]; then
+		echo "Installing compatible PHPUnit for use with PHP 8..."
+
+		PATCH=$(cat <<-EOF
+				"${WP_TESTS_DIR}/includes/phpunit7/MockObject/Builder/NamespaceMatch.php",
+				"${WP_TESTS_DIR}/includes/phpunit7/MockObject/Builder/ParametersMatch.php",
+				"${WP_TESTS_DIR}/includes/phpunit7/MockObject/InvocationMocker.php",
+				"${WP_TESTS_DIR}/includes/phpunit7/MockObject/MockMethod.php"
+			],
+			"exclude-from-classmap": [
+				"vendor/phpunit/phpunit/src/Framework/MockObject/Builder/NamespaceMatch.php",
+				"vendor/phpunit/phpunit/src/Framework/MockObject/Builder/ParametersMatch.php",
+				"vendor/phpunit/phpunit/src/Framework/MockObject/InvocationMocker.php",
+				"vendor/phpunit/phpunit/src/Framework/MockObject/MockMethod.php"
+		EOF
+		)
+
+		sed -i "/register-wp-cli-commands.php\"$/a , $(echo ${PATCH})" composer.json
```

Comment:
This seems somewhat brittle. If `composer.json` changes this line in any way:

https://github.com/ampproject/amp-wp/blob/31758e96a656e2cf722cce842bb301d7a6152763/composer.json#L88

Then `sed` will silently fail, no? 

Would it be better to include an actual `diff`/`patch` file that is applied with `patch`?

Reply:
> This seems somewhat brittle. If composer.json changes this line in any way:

Hmm yea, although it is meant as a temporary solution since Core should be upgrading to PHPUnit v9 soon. Nevertheless I've switched to using a diff instead to prevent the possibility of `sed` silently failing (see https://github.com/ampproject/amp-wp/pull/5475/commits/452b56dff87a9cabc8f744c126202a6bdfbdf421).